test/extended/bootstrap_user/bootstrap_user_login.go

```diff
@@ -0,0 +1,31 @@
+package bootstrap_user
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[Suite:openshift/smoke-4] The bootstrap user", func() {
```

Comment:
It's unclear to me what the exact label for this should be.  I went with [Suite:openshift/smoke-4] as my best guess.